go.mod

```diff
@@ -8,58 +8,57 @@ replace github.com/ramendr/ramen/api => ./api
 require (
 	github.com/aws/aws-sdk-go v1.44.289
 	github.com/backube/volsync v0.7.1
-	github.com/csi-addons/kubernetes-csi-addons v0.8.0
-	github.com/go-logr/logr v1.3.0
-	github.com/google/uuid v1.3.1
+	github.com/csi-addons/kubernetes-csi-addons v0.8.1-0.20240822090723-89d4c5b45a32
```

Comment:
I have to use specific commit from CSI because of the constants, that they forgot to expose in 0.9.0 and merged later. I don't like this situation also, if you can suggest a better solution - let's discuss it.

Reply:
I think this is fine for a temporary change until we get a new 0.9.1 wit this fix.

@Madhu-1 do you plan to release 0.9.1 soon?

Reply:
not at the moment (at least not in 3 weeks unless we have bugs that need a release)

Reply:
@ELENAGER You can get a higher commit than 0.9.0. The line should look like v0.9.1-...